source/helpers/orientation-validator.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-else-return */
+import {ERROR_CODES, ORIENTATION_ALL, ORIENTATION_LANDSCAPE, ORIENTATION_PORTRAIT} from '../constants';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)